android/keyframes-sample/build.gradle

```diff
@@ -20,7 +20,6 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
This is unused. The library gets imported to the sample by the `compile` statement in line 24.

I'm able to build and run the sample successfully with these changes.